mobile_english_preact/components/BannerText.jsx

```diff
@@ -0,0 +1,28 @@
+// eslint-disable-next-line no-unused-vars
+import { h } from 'preact';
+import TextHighlight from '../../shared/components/ui/TextHighlight';
+
+export default function BannerText( props ) {
+	const { weekdayPrepPhrase, currentDayName, numberOfDonors, campaignDaySentence, campaignParameters } = props;
+	return <div className="banner-text">
+		<p className="text__headline">
+			To all our readers in Germany.
+		</p>
+		<p>
+			It's a little awkward, so we'll get straight to the point: { weekdayPrepPhrase } { currentDayName } we humbly
+			ask you to protect Wikipedia's independence.
+			{ campaignDaySentence.getSentence() } We depend on donations averaging about € 23.83, but 99% of our readers don't give.
```

Comment:
See #310 for the changed `getSentence` code
```suggestion
			{ campaignDaySentence } We depend on donations averaging about € 23.83, but 99% of our readers don't give.
```